save-common/src/commonMain/kotlin/com/saveourtool/save/core/utils/PlatformUtils.kt

```diff
@@ -15,40 +15,6 @@ enum class CurrentOs {
     LINUX, MACOS, UNDEFINED, WINDOWS
 }
 
-/**
- * Atomic values
- */
-expect class AtomicInt(value: Int) {
```

Comment:
?

Reply:
they aren't used in code

Reply:
we dont use them in save cloud by the way?